fish.desktop

```diff
@@ -1,12 +0,0 @@
-[Desktop Entry]
```

Comment:
You merged/rebased this wrong. That file was added between you opening the PR and adding the second commit.